eclair-core/src/test/scala/fr/acinq/eclair/wire/internal/channel/ChannelCodecsSpec.scala

```diff
@@ -100,38 +100,6 @@ class ChannelCodecsSpec extends AnyFunSuite {
     assert(data_new == data_new2)
   }
 
-  test("backward compatibility DATA_NORMAL_COMPAT_03_Codec (roundtrip)") {
```

Comment:
I tried fixing this test, but it's a real pain because it uses one of our oldest codecs that has been updated years ago. I think we can safely drop it.